app/src/components/Modals/add-user-modal.tsx

```diff
@@ -94,8 +95,9 @@ const AddUserModal: FC<AddUserModalProps> = ({
         });
       } else {
         await inviteUser({
-          userId: res && res.data.data.userId!,
-          locode: userInfo.defaultCityLocode!,
+          name: res.data.data ? res.data.data.name! : data.name,
```

Comment:
Why res.data.data ? You can unwrap the API response in the Redux api.ts file like we do for the other API calls.